supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -115,55 +116,55 @@ In an IdM-AD trust setup, the `Default Trust View` is an ID view applied to AD u
 [discrete]
 [[identity-provider]]
 ==== image:images/yes.png[yes] identity provider (noun)
-*Description*: An identity provider (IDP) is a service that authenticates a user. Red Hat Single Sign-On is an IDP.
+*Description*: An _identity provider (IDP)_ is a service that authenticates a user. Red Hat Single Sign-On is an IDP.
 
 *Use it*: yes
 
 *Incorrect forms*:
 
-*See also*:
+*See also*: xref:identity-provider-federation[identity provider federation], xref:identity-provider-mappers[identity provider mappers]
 
 // RHSSO: Added "In Red Hat Single Sign-On, you can" and removed a few other words
 [discrete]
 [[identity-provider-federation]]
 ==== image:images/yes.png[yes] identity provider federation (noun)
-*Description*: In Red Hat Single Sign-On, you can delegate authentication to one or more IDPs. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IDP.
+*Description*: In Red Hat Single Sign-On, you can delegate authentication to one or more IDPs, and this process is referred to as _identity provider federation_. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IDP.
 
 *Use it*: yes
 
 *Incorrect forms*:
 
-*See also*:
+*See also*: xref:identity-provider[identity provider], xref:identity-provider-mappers[identity provider mappers]
 
 // RHSSO: General; kept as is
 [discrete]
 [[identity-provider-mappers]]
 ==== image:images/yes.png[yes] identity provider mappers (noun)
-*Description*: When doing IDP federation, you can map incoming tokens and assertions to user and session attributes. This helps you propagate identity information from the external IDP to your client requesting authentication.
+*Description*: When doing IDP federation, you can map incoming tokens and assertions to user and session attributes, which you refer to as _identity provider mappers_. This helps you propagate identity information from the external IDP to your client requesting authentication.
```

Comment:
which you refer to as "identity provider mappers"

Reply:
Changed